apps/router/src/guardian-ui/components/dashboard/gateways/GatewaysCard.tsx

```diff
@@ -16,9 +16,9 @@ import {
 import { ClientConfig, Gateway, ModuleKind } from '@fedimint/types';
 import { Table, TableColumn, TableRow } from '@fedimint/ui';
 import { useTranslation, formatEllipsized } from '@fedimint/utils';
-import { ModuleRpc } from '../../../types';
 import { ReactComponent as InfoIcon } from '../../../assets/svgs/info.svg';
 import { useGuardianAdminApi } from '../../../../context/hooks';
+import { ModuleRpc } from '../../../../types/guardian';
```

Comment:
nit: consider adding module aliases to reduce the number of dots in the path .../.../................................. vs @/...